python/pacemaker/_cts/tests/rollingupgradetest.py

```diff
@@ -0,0 +1,103 @@
+""" Test-specific classes for Pacemaker's Cluster Test Suite (CTS)
+"""
+
+__all__ = ["RollingUpgradeTest"]
+__copyright__ = "Copyright 2000-2023 the Pacemaker project contributors"
+__license__ = "GNU General Public License version 2 or later (GPLv2+) WITHOUT ANY WARRANTY"
+
+from pacemaker._cts.tests.ctstest import CTSTest
+from pacemaker._cts.tests.simulstartlite import SimulStartLite
+from pacemaker._cts.tests.simulstoplite import SimulStopLite
+from pacemaker._cts.tests.starttest import StartTest
+from pacemaker._cts.tests.stoptest import StopTest
+
+
+class RollingUpgradeTest(CTSTest):
```

Comment:
Let's just kill this test. I don't see any way to get rpm-dir, current-version, or previous-version into _env, and the lab is not guaranteed to work across versions since it relies on specific log patterns that sometimes change release to release. Not to mention it assumes RPM packaging. I think manually testing rolling upgrades is probably the best approach.

Reply:
Cut.